src/model/CoilSystemCoolingWaterHeatExchangerAssisted.cpp

```diff
@@ -45,6 +45,8 @@
 #include "AirLoopHVAC_Impl.hpp"
 #include "HeatExchangerAirToAirSensibleAndLatent.hpp"
 #include "HeatExchangerAirToAirSensibleAndLatent_Impl.hpp"
+#include "HeatExchangerDesiccantBalancedFlow.hpp"
+#include "HeatExchangerDesiccantBalancedFlow_Impl.hpp"
```

Comment:
The water one DOES not accept the HX Desicant. Remove from this class completely. See comment on https://github.com/NREL/OpenStudio-resources/pull/141#issuecomment-854467063 . Direct link to E+ source code where it clearly shows this isn't accepted. https://github.com/NREL/EnergyPlus/blob/b8f984adb5fe45cfebf838ab1fcf21d104e5091a/src/EnergyPlus/HVACHXAssistedCoolingCoil.cc#L647-L656

* [ ] TODO

Reply:
done in 194efe00e4adffee34c108c6fe378a1b29b8079f.